packages/analitics/index.ts

```diff
@@ -0,0 +1,49 @@
+import { observe } from "frontity";
+import Analytics from "./types";
+
```

Comment:
I think that apart from the named exports, I would also export the default export (containing `state` and `actions`) in this file.

Did you think about a reason not to do so? Do you think it can be misleading?

Reply:
I wanted to clearly show that this is not a package but a library, just looking at the code.